examples/flower_classifier/README.rst

```diff
@@ -0,0 +1,115 @@
+How To Train and Deploy Image Classifier with MLflow and Keras
+---------------------------------------------------------------
+
+In this example we demonstrate how to train and deploy image classification models with MLflow.
+We train a VGG16 deep learning model to classify flower species from photos using a `dataset
+<http://download.tensorflow.org/example_images/flower_photos.tgz>`_ available from `tensorflow.org
+<http://www.tensorflow.org>`_. Note that although we use Keras to train the model in this case,
+similar approach can be applied to other deep learning frameworks such as ``PyTorch``.
+
+The MLflow model produced by running this example can be deployed to any MLflow supported endpoints.
+All the necessary image preprocessing is packaged with the model. The model can therefore be applied
+to image data directly. All that is required in order to pass new data tot he model is to encode the
+image binary data as base64 encoded string in pandas DataFrame (standard interface for MLflow python
+function models). You can check included python scripts showing how to score model deployed to a
+REST api endpoint and also of batch scoring in Spark.
+
+In order to include a custom image pre-processing logic with the model, we define the model as a
+custom python function model wrapping around the underlying Keras model. The wrapper provides
+necessary preprocessing to convert input data into a multidimensional arrays expected by the
+Keras model. The preprocessing logic is stored with the model as a code dependency. Here is an
+example of the output model directory layout:
+
+.. code:: bash
+
+   tree model
+
+::
+
+    model
+    ├── MLmodel
+    ├── code
+    │   └── image_pyfunc.py
+    └── data
+        └── tmpf7mnjr6w
```

Comment:
Why does this directory get a weird name? we could just call it keras_model or something